php/datasource/class-fieldmanager-datasource-user.php

```diff
@@ -156,4 +219,25 @@ public function get_edit_link( $value ) {
             esc_html__( 'Edit', 'fieldmanager' )
         );
     }
+    
+    /**
+     * Sanitize the value based on store_property.
+     * @param int|string $value
+     * @return int|string
+     */
+    private function sanitize_value( $value ) {
+    	switch ( $this->store_property ) {
+    		case 'ID':
+    			$value = intval( $value );
+    			break;
+    		case 'user_email':
+    			$value = sanitize_email( $value );
+    			break;
+    		default:
+    			$value = sanitize_text_field( $value );
+    			break;
+    	}
+    	
+    	return $value;
+    }
```

Comment:
Niiiiice. I would make this `protected` instead of private so classes extending this one can use it.